### PR TITLE
Fix up navigation on the home container and create proper back navigation on the display hymns screen.

### PR DESCRIPTION
**Pull request by skywalkerdude**
Fix up navigation on the home container and create proper back navigation on the display hymns screen. Getting closer to an MVP! Also there's finally enough functionality to write some UI tests

### CODE REVIEW

File: Hymns/Display/DisplayHymnView.swift [hunk 1]
```diff
@@ -3,47 +3,35 @@ import Resolver
...
-                    HStack {
-                        NavigationLink(destination: HomeContainerView()) {
-                            Image(systemName: "xmark").accentColor(Color.black)
-                        }
-                        Spacer()
-                        if self.viewModel.favorited == true {
-                        }
-                        //TODO: Favorited needs to actually do something when clicked other than just say "favorited"
-                        Spacer()
-                        Button(action: {self.viewModel.toggleFavorited()}) {
```
Comment (finben) [inline Hymns/Display/DisplayHymnView.swift:21]: Any reason in particular you nixed my heart button from working? I can put it back in later but just wanted to check

Comment (finben) [inline Hymns/Display/DisplayHymnView.swift:21]: Any reason in particular you nixed my heart button from working? I can put it back in later but just wanted to check

File: Hymns/Display/DisplayHymnView.swift [hunk 2]
```diff
@@ -3,47 +3,35 @@ import Resolver
...
-        }.navigationBarTitle("", displayMode: .inline).navigationBarHidden(true)
-    }
-
-    init(viewModel: DisplayHymnViewModel) {
-        self.viewModel = viewModel
+            Spacer()
+        }
+        .hideNavigationBar()
+        .onAppear {
+            self.viewModel.fetchHymn()
```
Comment (finben) [inline Hymns/Display/DisplayHymnView.swift:33]: This is cool. So it does the fetch on appear. Nice

File: Hymns/Display/DisplayHymnView.swift (hunk 1 above)
Comment (skywalkerdude) [reply Hymns/Display/DisplayHymnView.swift:21]: ah oops. Might have been an oversight on my part. Could you add it back later, when you add the favorites with realm?

Comment (skywalkerdude) [reply Hymns/Display/DisplayHymnView.swift:21]: ah oops. Might have been an oversight on my part. Could you add it back later, when you add the favorites with realm?

File: Hymns/Display/DisplayHymnView.swift (hunk 2 above)
Comment (skywalkerdude) [reply Hymns/Display/DisplayHymnView.swift:33]: Yeah, haha. That way it won't be making like hundreds of requests to my free heroku server and overloading it 😂

**Pull request merged**